pptx_blueprint/__init__.py

```diff
@@ -37,7 +37,27 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
             label (str): label of the placeholder (without curly braces)
             filename (path-like): path to an image file
         """
-        pass
+        shapes_to_replace = self._find_shapes(label)
+        if not shapes_to_replace:
+            return
+        if isinstance(filename, str):
+            filename = pathlib.Path(filename)
+        if not filename.is_file():
+            raise FileNotFoundError(f"The file does not exist: {filename}")
+        img_file = open(filename, "rb")
+        old_shape: BaseShape
+        for old_shape in shapes_to_replace:
+            slide_shapes = old_shape._parent
+            slide_shapes.add_picture(
+                image_file=img_file,
+                left=old_shape.left,
+                top=old_shape.top,
+                width=old_shape.width,
+                height=old_shape.height
+            )
+            # Removing shapes is performed at the lxml level.
+            # The `element` attribute contains an instance of `lxml.etree._Element`.
+            slide_shapes.element.remove(old_shape.element)
```

Comment:
Yikes! We should really not deal with xml ourselves. We should open a RP in pptx to get this feature implemented. For the time being, we should either 1) isolate this in a private method marked deprecated to prevent excessive use, or 2) decorate the pptx presentation object.